src/sdk/namespace_nb.js

```diff
@@ -129,6 +129,7 @@ class NamespaceNB {
             active_triggers,
             operation
         });
+        params = _.omit(params, 'force_md5_etag');
```

Comment:
why is this omitted from here? we want to also use it inside...

Reply:
for this PR we will only use this flag for NSFS. we can add it in nb buckets later